screen/setup_mae_from_com.py

```diff
@@ -69,8 +69,8 @@ def read_com(filename):
     # Remove all extra 0's from comp and chig.
     # Up to 4 COMP atoms are given per line. If 3 or less are given on a line,
     # the remaining columns are 0's. Delete those meaningless 0's.
-    comp = filter(lambda x: x != 0, comp)
-    chig = filter(lambda x: x != 0, chig)
+    comp = list(filter(lambda x: x != 0, comp))
```

Comment:
Similarly to map(), usually list comprehensions are better than wrapping list() around filter(), especially when using a lambda function like this. This can be done like this
```suggestion
    comp = [x for x in comp if x != 0]
```